tools/GitHubTools.psm1

```diff
@@ -85,7 +85,7 @@ function GetGitHubHeaders
         $Accept
     )
 
-    $headers = @{}
+    $headers = @{"Content-Type" = "UTF-8"}
```

Comment:
The right fix for this is for us to move over to [PowerShellForGitHub](https://github.com/microsoft/PowerShellForGitHub), but:

```suggestion
    $headers = @{"Content-Type" = "application/json; charset=utf-8"}
```

Reply:
See also:
- [MDN: Content-Type header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type)
- [MDN: Common MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)

Reply:
OMG, thank you.